lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1640,17 +1641,27 @@ def _deploy(
             deployment_operation = deployments.begin_create_or_update(
                 **deployment_parameters
             )
-            while True:
+            timer = create_timer()
+            while timer.elapsed(False) < 3600:  # 60 minutes
```

Comment:
Move the value to be configurable in Azure platform level, name it like `provisioning_timeout` or better name.